.gitignore

```diff
@@ -12,3 +12,4 @@
 *.out
 
 bin/
+.idea/
```

Comment:
Would you please move this to your [global gitignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/)?

Reply:
Ok sure